includes/Install.php

```
@@ -19,4 +20,28 @@ public function run() {
+     *
+     * @return void
+     */
+    private function create_tables() {
+        global $wpdb;
+
+        require_once ABSPATH . 'wp-admin/includes/upgrade.php';
+
+        $charset_collate = $wpdb->get_charset_collate();
+        $table_name      = $wpdb->prefix . 'texty_sms_stat';
+
+        $sql = "CREATE TABLE IF NOT EXISTS {$table_name} (
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$sql'. (undefined)

(UnusedLocalVariable)